packages/amplication-server/src/core/resource/resourceBtm.service.ts

```diff
@@ -178,26 +156,27 @@ export class ResourceBtmService {
   }
 
   generatePromptForBreakTheMonolith(resource: ResourceDataForBtm): string {
-    const entityIdNameMap = resource.entities.reduce((acc, entity) => {
-      acc[entity.id] = entity.name;
-      return acc;
-    });
+    const entityNameToRelatedFieldsMap = resource.entities.reduce(
+      (acc, entity) => {
+        const relationFields = entity.versions[0].fields.filter(
+          (field) => field.dataType === EnumDataType.Lookup
+        );
+        const entityFields = relationFields.length
+          ? [...new Set(relationFields.map((field) => field.name))]
```

Comment:
Just one question here about mapping to `field.name`  :
What if the field name is not the name of the remote relation? E.g. I have a field named "manager" that is a relation to the "Employee" table?

Reply:
good one!